backend/tests/verify_fixtures.py

```
@@ -0,0 +1,24 @@
+from items.models import Location, OwnedItem
+from users.models import User
+
+users = User.objects.filter(clerk_id__startswith="user_test")
+locations = Location.objects.all()
+items = OwnedItem.objects.all()
+
+print(f"\n=== Fixture Verification ===")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)